modules/monitoring-managing-core-platform-alerting-rules.adoc

```diff
@@ -0,0 +1,28 @@
+// Module included in the following assemblies:
+//
+// * monitoring/managing-alerts.adoc
+
+:_content-type: CONCEPT
+[id="managing-core-platform-alerting-rules_{context}"]
+= Managing alerting rules for core platform monitoring
+
+:FeatureName: Creating and modifying alerting rules for core platform monitoring
+include::snippets/technology-preview.adoc[leveloffset=+1]
+
+{product-title} {product-version} monitoring ships with a large set of default alerting rules for platform metrics.
+As a cluster administrator, you can customize this set of rules in two ways:
+
+* Modify the settings for existing platform alerting rules by adjusting thresholds or by adding and modifying labels.
+For example, to help you route and triage issues flagged by an alert, you can change the `severity` label for an alert from `warning` to `critical`.
+
+* Define and add new custom alerting rules by constructing a query expression based on core platform metrics in the `openshift-monitoring` namespace.
+
+.Core platform alerting rule considerations
+
+* New alerting rules must be based on the default {product-title} monitoring metrics.
+
+* You can only add and modify alerting rules. You cannot create new recording rules or modify existing recording rules.
+
+* If you modify existing platform alerting rules, your modifications are not reflected in the Prometheus alerts API. 
+Therefore, currently dropped alerts still appear in the {product-title} web console even though they were forwarded to Alertmanager. 
+Additionally, any modifications made to alerts such as a changed severity label do not appear in the web console.
```

Comment:
```suggestion
* If you modify existing platform alerting rules though an `AlertRelabelConfig` object, your modifications are not reflected in the Prometheus alerts API. 
Therefore, any dropped alerts still appear in the {product-title} web console even though they are no longer forwarded to Alertmanager. 
Additionally, any modifications made to alerts such as a changed severity label do not appear in the web console.
```
Sorry about the late correction on this. I had to have a closer look myself to confirm how this works. Based on https://github.com/openshift/enhancements/pull/1030/files#r811793390 and https://prometheus.io/docs/prometheus/latest/configuration/configuration/#alert_relabel_configs.

Reply:
Changed to read:

```
* If you modify existing platform alerting rules by using an `AlertRelabelConfig` object, your modifications are not reflected in the Prometheus alerts API. 
Therefore, any dropped alerts still appear in the {product-title} web console even though they are no longer forwarded to Alertmanager. 
```